test/CheckedCRewriter/function_tests.c

```diff
@@ -0,0 +1,52 @@
+// Tests for Checked C rewriter tool.
```

Comment:
How about tests for parameters that take function pointer types, returning a function pointer type, and arrays of function pointer types?